crates/astria-sequencer-relayer/src/finalization_pipeline/pipeline_item.rs

```diff
@@ -0,0 +1,87 @@
+use astria_sequencer_types::SequencerBlockData;
+use tendermint::Hash;
+
+use super::{
+    Head,
+    HeadCandidate,
+};
+
+#[derive(Clone, Copy, Default, Debug)]
+enum State {
```

Comment:
please add comment explaining this

Reply:
also remove `Default`, don't think there's a natural default for this

Reply:
I believe it is in fact `Fork` since requirements for this code was that blocks coming from sequencer always point to [forks of] the canonical chain and always arrive in sequential order relative to height